packaging/helm/trento-server/templates/prometheus_configmap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-prometheus-configmap
+data:
+  prometheus.yml: |-
```

Comment:
Most probably we will need to change this on the road. But I guess it is totally fine by now